src/components/common/RegAddEditOrgPerson.vue

```diff
@@ -78,7 +78,9 @@
                   filled
                   class="item first-name"
                   label="First Name"
-                  :rules="enableRules ? Rules.FirstNameRulesFirms : []"
+                  :rules="enableRules ?
+                   (isCompletingParty ? Rules.FirstNameRulesFirms.slice(1) :
+                   Rules.FirstNameRulesFirms) : []"
```

Comment:
1. This is too much logic for the template, but...
2. This is not a good way to implement this as it assumes that the first rule (which you are omitting) is for first name. However, someone could change the rules (in the other file) and would not know to update this file, causing a bug.

I recommend creating a different rule for Completing Party. You could add it to the same file as the other rule.

Reply:
Thanks for the fix.